lib/ansible/plugins/doc_fragments/ns1.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Matthew Burtless <mburtless@ns1.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    # NS1 doc fragment
+    DOCUMENTATION = r'''
+options:
+  apiKey:
+    description:
+      - Unique client api key that can be created via the NS1 portal.
+    required: true
+
+requirements:
+  - python >= 2.7
+  - ns1 >= 0.9.19
```

Comment:
`ns1-python`?

Reply:
The name in PyPi is `ns1-python` but the namespace is `ns1`.  Wasn't clear to me which we should be listing here (or if it matters), but I think you're right that package name probably makes more sense.

Reply:
true.. I think PyPI package name makes more sense.